iGCS/UserInterface/ViewControllers/GCSMapViewController.m

```diff
@@ -157,6 +165,29 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interface
     return YES;
 }
 
+- (void)dealloc {
+    [[NSNotificationCenter defaultCenter] removeObserver:self];
+}
+
+#pragma mark - 
+#pragma mark Handle NSnotifications for UI updates
+
+// These will get called only when there is a change event
+// instead of at every tick of the telemetry data
+- (void)craftMavModeDidChange {
+    // Update armed status labels
+    if ([GCSDataManager sharedInstance].craft.isArmed) {
```

Comment:
3 lines become 7 to avoid the ternary hate. /sheds tear

I'll console myself that it could be worse if we weren't using 1TBS.